src/libecalc/common/stream.py

```diff
@@ -15,6 +15,7 @@ class Config:
         alias_generator = to_camel_case
         allow_population_by_field_name = True
 
+    name: Optional[str]
     rate: TimeSeriesRate
```

Comment:
I guess we need either a (unique) name or an id to attach a stream to a compressor stage when we later allow for streams going into/out of any compressor stage. 


Also consider including temperature here and the full fluid model/composition, not just the density.

Reply:
I agree, the plan is to implement that in another PR. I'm thinking a unique name for a stream within a component makes sense.